File: sdk/client/java/openapi-test/src/test/java/opensdk/sdk/apis/klaytnDebug/logging/DebugVerbosityByNameTest.java
package opensdk.sdk.apis.klaytnDebug.logging;

import opensdk.sdk.apis.constant.UrlConstants;
import opensdk.sdk.models.DebugVerbosityByNameResponse;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import org.klaytn.OpenSDK;

import java.io.IOException;

@DisplayName("Debug RPC Test")
public class DebugVerbosityByNameTest {
    private final OpenSDK sdk = new OpenSDK(UrlConstants.SERVER_URL);

    @Test
    @DisplayName("RPC debug_verbosityByName")
    void whenRequestValid_ThenCall200ResponseReturns() throws IOException {
        String name = "API";
        int level = 3;

        DebugVerbosityByNameResponse response = sdk.debug.verbosityByName(name, level).send();
        response.getResult();
    }
}

File: web3rpc/sdk/client/javascript/openapi-test/test/debug/logging/DebugVerbosityByName.test.js
const OpenSdk = require("@klaytn/web3rpc");
const { expect } = require("@jest/globals");
const { RPC } = require("../../constant");

const sdk = new OpenSdk.DebugApi(new OpenSdk.ApiClient(RPC));

describe('debug_verbosityByName API', () => {
    test.skip('should return debug_verbosityByName', (done) => {

        let callbackOne = function (error, data, response) {
            expect(error).toBeNull();
            expect(data).toBeDefined()
            expect(data).toBe(null);
            done();
        };

        const name = "name";
        const level = 3;

        sdk.verbosityByName(name, level, {}, callbackOne);
    });
});

